CHANGELOG.md

```diff
@@ -9,6 +9,33 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
+-
+
+### Fixes
+
+-
+
+### Internal Changes
+
+-
+
+### Deprecations
+
+-
+
+### Removals
+
+-
+
+### Security
+
+-
+
+
+## [0.1.55] - 2022-09-15
+
+### Enhancements
+
```

Comment:
Should we have a release note for the limits support @jhrozek added since that's technically going out the door this release?

Reply:
Fixed in https://github.com/ComplianceAsCode/compliance-operator/pull/133